src/hotspot/share/runtime/javaThread.cpp

```diff
@@ -109,6 +109,10 @@
 #include "jfr/jfr.hpp"
 #endif
 
+#ifdef _AIX
+#include <alloca.h>
+#endif
+
```

Comment:
Could these conditionals be included in globalDefinitions_xlc.hpp instead?

Reply:
In principle the `#include <alloca.h>` could be included in globalDefinitions_xlc.hpp. But alloca.h implements it with a
`#undef  alloca`
`#define alloca(size)   __builtin_alloca (size)`
If this new global define does not introduce new trouble (even in future coding) when it is seen in every source, we can go this way.
Are there any obstacles from anyone else?

Reply:
At least the whole jdk actually builds with `#include <alloca.h>` in globalDefinitions_xlc.hpp